content/en/agent/kubernetes/apm.md

```diff
@@ -44,13 +44,16 @@ To enable trace collection with your Agent, follow the instructions below:
         #
         enabled: true
     ```
- - Then, upgrade your Datadog Helm chart using the following command : `helm upgrade -f datadog-values.yaml <RELEASE NAME> stable/datadog` (don't forget to set the API key in the yaml file)
+
+    **Note**: In advanced use cases on Windows, you may need to add the line `targetSystem: windows` to the top of your `values.yaml`.
```

Comment:
There are basically two options:
- You always set `targetSystem` in your `values.yaml` and use one `values.yaml` for Windows and one for Linux and all the commands remain identical
- You never set `targetSystem` in your `values.yaml` and you ALWAYS need to put `--set targetSystem=windows`

With the second option, if you forget to set `--set targetSystem=windows`, you may end up replacing a Windows deployment by a Linux deployment (as targetSystem defaults to `Linux` if not set)